lib/utils/file_uploader.dart

```diff
@@ -81,6 +82,19 @@ class FileUploader {
       }
       _pollBackgroundUploadStatus();
     }
+    Bus.instance.on<LocalPhotosUpdatedEvent>().listen((event) {
+      if (event.type == EventType.deletedFromDevice ||
+          event.type == EventType.deletedFromEverywhere) {
+        removeFromQueueWhere((file) {
+          for (final updatedFile in event.updatedFiles) {
+            if (file.generatedID == updatedFile.generatedID) {
```

Comment:
when is the `generatedID` added a file?

Reply:
It's auto generated during insertion into the DB. So it unique per row.